assignment/tool/src/webapp/js/studentViewSubmission.js

```diff
@@ -1,4 +1,8 @@
 var ASN_SVS = ASN_SVS || {};
+var ASN = ASN || {};
+var ASN_TS_API = ASN_TS_API || {};
```

Comment:
Any concerns with these being var and not let?

Reply:
Hi, @profmikegreene, this lines use global variables defined and reused across some files so should sadly keep as var